packaging/nmstate.spec

```diff
@@ -26,7 +26,7 @@ Summary:        nmstate Python 3 API library
 Requires:       NetworkManager-libnm >= 1:1.22.10
 # Use Recommends for NetworkManager because only access to NM DBus is required,
 # but NM could be running on a different host
-Recommends:     NetworkManager
+Requires:     NetworkManager
```

Comment:
As the above comments states, please still use `Recommends` for NM.

Reply:
The above statement is now false. We now need full access to NM AFAIK, because we are now not using dbus directly for checkpoint support. Could we use nmstate without NM installed? Am I missing something? Thanks!

Reply:
nmstate just needs the dbus socket to work with NM, not NM itself.
kubernetes-nmstate once worked in an isolates manner, where they deployed k-nmstate and nmstate on a container and used the dbus socket to communicate with NM. As this introduces several limitations (for the parts that NM is not involved) the approach was dropped.

Although I think no one else uses it like this anymore, this may still be a valid deployment. You will need to explicitly drop support for such a deployment and with it this specific dependency type on NM.

Reply:
Okay, I agree with this. Thanks for explaining :-)

Reply:
Thanks for the comments. I've changed that part back.